src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -51,7 +51,7 @@ const ProfilePicture = ({
                   style={styles.profileImage(size, width)}
                   source={{
                     uri: profilePicPath ? `${profilePicPath}` : DEFAULT_PROFILE_PIC_PATH,
-                    cache: 'cacheOnly'
+                    cache: 'web'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `cache: 'cacheOnly'` to `cache: 'web'` might have performance implications. The 'web' option will make a network request and update the cache if the image has changed. This could lead to unnecessary network requests if the profile picture doesn't change frequently. Consider reverting this change if profile pictures are not updated often.
```diff
- cache: 'web'
+ cache: 'cacheOnly'
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->